eclipse-scout-core/src/table/TableHeader.ts

```diff
@@ -310,7 +311,11 @@ export class TableHeader extends Widget implements TableHeaderModel {
       return column.headerTooltipText;
     }
     let $text = $col.children('.table-header-item-text');
-    if ($text.isContentTruncated() || ($col.width() + $col.position().left) > $col.parent().width()) {
+    let textBounds = graphics.offsetBounds($text);
+    let containerBounds = this._offsetBoundsWithoutMenuBar();
+    let textInView = textBounds.right() < containerBounds.right() && textBounds.x >= containerBounds.x;
```

Comment:
`textBounds.right() < containerBounds.right()` --> shouldn't this use the `<=` operator as well?